src/EventTeachingService.cpp

```diff
@@ -419,7 +433,7 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
             for (byte i = 1; i <= module_config->EE_NUM_EVS; i++)
             {
               controller->sendMessageWithNN(OPC_EVANS, i, module_config->getEventEVval(index, i));
-              delay(10);
+//              delay(10);
```

Comment:
If it works without this delay, please remove this line. No need for such commented code.

Reply:
Done

Reply:
Should the various delays have names?  Might make them easier to modify at a later date.

Reply:
@dpharris The delay was only there so as not to overload receiving modules with a mini storm of messages. It is now unnecessary here because CAN2515 puts a delay in between sending consecutive messages.